src/switch/switch.component.ts

```diff
@@ -60,17 +52,27 @@ export class SwitchChange {
 	selector: "ibm-switch",
 	template: `
 		<input
-			class="bx--toggle"
+			[ngClass]="{
+				'bx--toggle': true,
```

Comment:
We normally leave class names that are always true in `class`, so that would bring back the deleted line and remove this one.